backend/terry/handlers/info_handler.py

```diff
@@ -131,6 +131,30 @@ def get_user(self, user):
 
         return BaseHandler.format_dates(user, fields=["end_time"])
 
+    @Validators.contest_started
```

Comment:
```suggestion
    @Validators.contest_started
    @Validators.register_user_ip
```

Haven't tested it yet.

Reply:
It seems to work.